nilearn/input_data/nifti_spheres_masker.py

```diff
@@ -36,8 +38,14 @@ def _apply_mask_and_get_affinity(seeds, niimg, radius, allow_overlap,
 
         X = masking._apply_mask_fmri(niimg, mask_img)
     else:
+        #Adding  warning message 
+        if np.isnan(np.sum(_safe_get_data(niimg))):
+            warnings.warn('The imgs you have fedded into fit_transform()' 
```

Comment:
Change the warning message to "niimg fed into fit_transform() contains NaNs which will be converted to zeros"

Reply:
I would keep it ```imgs``` rather than ```niimg``` as per the doc string at fit_transform.

Reply:
Okay I will update my comment.